example/90-shoot.yaml

```
@@ -293,8 +293,10 @@ spec:
   #   ignoreTaints: 
   #     - "node.kubernetes.io/memory-pressure"
   #     - "node.kubernetes.io/disk-pressure"
+  #   ignoreDaemonsetsUtilization: false
   #   newPodScaleUpDelay: 10s
   #   maxEmptyBulkDelete: 10
+  #   verbosity: 2
```

Maintain the same ordering as defined in `types_shoot.go` file.

```suggestion
  #   ignoreDaemonsetsUtilization: false
  #   verbosity: 2
```